es-vue-base/src/lib-components/EsPopover.vue

```diff
@@ -212,14 +214,9 @@ export default {
     }
 
     .popover-body {
-        .btn, .btn:hover, .btn:active {
+        .btn, .btn:hover, .btn:active, a {
             color: variables.$white;
```

Comment:
For buttons, this color happens to be correct for the `dark-bg` variant, but it prevents that variant's outline version from getting the correct non-white hover and active colors. I think we should remove all the `.btn` classes from this rule and just leave `a`.

<img width="323" alt="Screen Shot 2024-05-07 at 5 01 39 PM" src="https://github.com/EnergySage/es-ds/assets/1350363/f3f78a4b-7810-420f-8010-8874518d58b8">